flow2ml/Flow.py

```diff
@@ -202,3 +204,15 @@ def getDataset(self,img_dimensions,train_val_split):
     self.update_progress( 100/100.0,"Created Datasets" ) 
 
     return self.dataset
+
+  def deployTensorflowModels(self):
+    '''Deploy conversion from tensorflow model to tensorflowjs or tflite model'''
+
+    if(self.model['tfjs']==True):
+      tfjs.converters.save_keras_model(model, 'models')
```

Comment:
Where do you get model here?

Reply:
line number 212 is correct when we get the model as an input parameter.
The function was correct

Reply:
> line number 212 is correct when we get the model as an input parameter.
> The function was correct

From where do we get the input model? where is it stored? 
and where should the output tfjs/tflite model be stored?

Reply:
@ainy-123 you get input model from user when he/she train model. For now just specify model. @yvkrishna can you specify about output part?

Reply:
@ainy-123, In general, we create all folders in the root folder. 

the model needs to be stored in the trained_models folder. In that folder, we can save the python model, tflite model, tfjs models.